README.md

```diff
@@ -17,10 +17,9 @@ If you want to use OpenBLAS, install it in any directory:
   make
 ```
```

Comment:
While we're here, could we add a comment that CGO_LDFLAGS can be set as an environment variable? I would guess we'll eventually have people who won't understand that from the current documentation. A high-level comment along the lines of "The cgo package provides bindings to c-backed BLAS packages. blas/cgo needs the CGO_LDFLAGS environment variable to point to the blas installation. More information can be found in the cgo documentation [link to http://golang.org/cmd/cgo/] "